### ISSUE TITLE
`tsc` **error TS2307**

### ISSUE THREAD

**Issue by groteck** [rmallols/ng-book2#1]
Hi I follow the explanations described on the book and when I run the first time the command `tsc` or `npm run tsc` without any arguments an error is through:

```bash
app.ts(1,27): error TS2307: Cannot find module '@angular/platform-browser-dynamic'.
app.ts(2,27): error TS2307: Cannot find module '@angular/core'.
``` 

I read the **WARNING** that says that if you pass some file to the `tsc` command, you can see this error but I use the command how was indicated on the book.

**Comment by ted-piotrowski** [comment]
The Ng-Book 2 and these code examples are not in sync. The book uses the releases candidate angular packages instead of the beta release packages. The best way to resolve this issue is to replace the package.json, tsconfig.json with the examples here: https://angular.io/docs/ts/latest/quickstart.html#!#add-config-files

You will also need to create a typings.json file and place the systemjs.config.js file in the resources/ folder.

In summary, these code examples are for a previous version of the book and thus are no longer in sync with the book contents.
